pkgs/applications/version-management/gitlab/data.json

```diff
@@ -1,15 +1,15 @@
 {
-  "version": "17.5.2",
-  "repo_hash": "08mx9xbl6cagzp36qwzz1zab5w8y2x4yhy3x5hqv1qwbrjrlcxjd",
+  "version": "17.5.4",
+  "repo_hash": "sha256-hKG9IZthAJc99wogi8K/4PgM5zTs525jKDZP9k6Clt0=",
```

Comment:
The hash is now in a different format as nix-universal-prefetch threw an error and I generated the hash with `nix flake prefetch`. I'll soon prepare a PR to switch the update script to `nix flake prefetch` as nix-univeral-prefetch is also dropped because it's not maintainted anymore.